src/index.js

```diff
@@ -89,7 +89,7 @@ class AppServer {
         this.app.use(bodyParser.urlencoded({ extended: true }));
         this.app.use(bodyParser.json());
         this.app.use(compression());
-        this.app.use('/public', express.static(path.join(__dirname, 'static')));
+        this.app.use(`${conf.GKB_BASE_PATH || ''}/public`, express.static(path.join(__dirname, 'static')));
```

Comment:
GKB_BAsE_PATH has default value '' anyway, doesn't it?

Reply:
ya but this is just a precaution for if someone passes it in as undefined or null instead